.circleci/config.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - run:
           name: install python dependencies
           command: |
-            curl -sSL https://install.python-poetry.org | POETRY_VERSION=1.2.0 POETRY_HOME=$HOME/.poetry/env python3 -
+            curl -sSL https://install.python-poetry.org | POETRY_VERSION=1.1.8 POETRY_HOME=$HOME/.poetry/env python3 -
```

Comment:
This was no linger working in circle CI. I updated it as per the error I was seeing.

```
Retrieving Poetry metadata

This installer is deprecated, and scheduled for removal from the Poetry repository on or after January 1, 2023.
See https://github.com/python-poetry/poetry/issues/6377 for details.

You should migrate to https://install.python-poetry.org instead, which supports all versions of Poetry, and allows for `self update` of versions 1.1.7 or newer.
Instructions are available at https://python-poetry.org/docs/#installation.

Without an explicit version, this installer will attempt to install the latest version of Poetry.
This installer cannot install Poetry 1.2.0a1 or newer (and installs will be unable to `self update` to 1.2.0a1 or newer).

To continue to use this deprecated installer, you must specify an explicit version with --version <version> or POETRY_VERSION=<version>.
Alternatively, if you wish to force this deprecated installer to use the latest installable release, set GET_POETRY_IGNORE_DEPRECATION=1.

Version 1.2.0 is not supported by this installer! Please specify a version prior to 1.2.0a1 to continue!

Exited with code exit status 1
```